frontend/src/old-pages/Configure/Configure.tsx

```diff
@@ -301,22 +308,26 @@ function Configure() {
   useWizardSectionChangeLog()
 
   return (
-    <Layout contentType="form">
-      <StopDialog clusterName={clusterName} />
-      <SpaceBetween direction="vertical" size="l">
-        <BreadcrumbGroup
-          items={[
-            {text: 'Clusters', href: '#clusters'},
-            // @ts-expect-error TS(2741) FIXME: Property 'href' is missing in type '{ text: string... Remove this comment to see the full error message
-            {text: editing ? 'Update' : 'Create'},
-          ]}
+    <Layout
+      contentType="form"
+      breadcrumbs={
+        <Breadcrumbs
           onClick={clearStateAndCloseWizard}
+          pages={[
```

Comment:
todo: memoize this array with `useMemo` since you are using a variable here. For all the other places a simple extract from component + `const` is sufficient

Reply:
Done (and I also centralized the implementation so this snippet got simpler), I used `useMemo` to avoid re-renders.
Thanks!